ui/resetpassword.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE HTML>
```

Comment:
Thanks Miguel. Would it be fine to have requests from such url redirected to a error page?